src/overlay110/ov110_021D0D80.c

```diff
@@ -697,235 +698,49 @@ static void ov110_021D19B0 (UnkStruct_ov110_021D0F78 * param0)
-    add r1, r0, #0
-    mov r0, #0x4b
-    lsl r0, r0, #2
-    ldr r0, [r5, r0]
-    mov r2, #0xff
-    bl sub_02030698
-    add r2, r0, #0
-    add r0, r5, #0
-    mov r1, #0
-    bl ov110_021D13F0
-    mov r2, #0
-    str r2, [sp]
-    mov r1, #1
-    str r1, [sp, #4]
-    mov r0, #2
-    str r0, [sp, #8]
-    str r2, [sp, #0xc]
-    str r2, [sp, #0x10]
-    str r2, [sp, #0x14]
-    str r1, [sp, #0x18]
-    add r1, r5, #0
-    add r0, r5, #0
-    add r1, #0x30
-    mov r2, #0x2a
-    mov r3, #0x70
-    bl ov110_021D1324
-    mov r0, #0x4a
-    lsl r0, r0, #2
-    ldr r0, [r5, r0]
-    bl SaveData_GetBattleFrontier
-    add r4, r0, #0
-    ldrb r0, [r5, #5]
-    bl sub_0205E630
-    add r6, r0, #0
-    ldrb r0, [r5, #5]
-    bl sub_0205E630
-    bl sub_0205E6A8
-    add r2, r0, #0
-    add r0, r4, #0
-    add r1, r6, #0
-    bl sub_02030698
-    add r2, r0, #0
-    add r0, r5, #0
-    mov r1, #0
-    bl ov110_021D13F0
-    mov r1, #0
-    str r1, [sp]
-    mov r0, #1
-    str r0, [sp, #4]
-    mov r0, #2
-    str r0, [sp, #8]
-    str r1, [sp, #0xc]
-    str r1, [sp, #0x10]
-    str r1, [sp, #0x14]
-    add r1, r5, #0
-    str r0, [sp, #0x18]
-    add r0, r5, #0
-    add r1, #0x30
-    mov r2, #0x29
-    mov r3, #0xe0
-    bl ov110_021D1324
-    add r0, r5, #0
-    add r0, #0x30
-    bl Window_ScheduleCopyToVRAM
-    add r1, r5, #0
-    add r0, r5, #0
-    add r1, #0x40
-    mov r2, #0x1f
-    mov r3, #1
-    bl ov110_021D13CC
-    ldrb r0, [r5, #5]
-    bl sub_0205E608
-    add r1, r0, #0
-    mov r0, #0x4b
-    lsl r0, r0, #2
-    ldr r0, [r5, r0]
-    mov r2, #0xff
-    bl sub_02030698
-    add r2, r0, #0
-    add r0, r5, #0
-    mov r1, #0
-    bl ov110_021D13F0
-    mov r2, #0
-    str r2, [sp]
-    mov r1, #1
-    str r1, [sp, #4]
-    mov r0, #2
-    str r0, [sp, #8]
-    str r2, [sp, #0xc]
-    str r2, [sp, #0x10]
-    str r2, [sp, #0x14]
-    str r1, [sp, #0x18]
-    add r1, r5, #0
-    add r0, r5, #0
-    add r1, #0x40
-    mov r2, #0x2a
-    mov r3, #0x70
-    bl ov110_021D1324
-    mov r0, #0x4a
-    lsl r0, r0, #2
-    ldr r0, [r5, r0]
-    bl SaveData_GetBattleFrontier
-    add r4, r0, #0
-    ldrb r0, [r5, #5]
-    bl sub_0205E680
-    add r6, r0, #0
-    ldrb r0, [r5, #5]
-    bl sub_0205E680
-    bl sub_0205E6A8
-    add r2, r0, #0
-    add r0, r4, #0
-    add r1, r6, #0
-    bl sub_02030698
-    add r2, r0, #0
-    add r0, r5, #0
-    mov r1, #0
-    bl ov110_021D13F0
-    mov r1, #0
-    str r1, [sp]
-    mov r0, #1
-    str r0, [sp, #4]
-    mov r0, #2
-    str r0, [sp, #8]
-    str r1, [sp, #0xc]
-    str r1, [sp, #0x10]
-    str r1, [sp, #0x14]
-    add r1, r5, #0
-    str r0, [sp, #0x18]
-    add r0, r5, #0
-    add r1, #0x40
-    mov r2, #0x29
-    mov r3, #0xe0
-    bl ov110_021D1324
-    add r5, #0x40
-    add r0, r5, #0
-    bl Window_ScheduleCopyToVRAM
-    add sp, #0x1c
-    pop {r3, r4, r5, r6, pc}
+    u8 v0;
+    int v1;
+    u32 v2;
+
+    v0 = 0;
+    Window_FillTilemap(&param0->unk_10[v0], 0);
```

Comment:
```suggestion
    Window_FillTilemap(&param0->unk_10[0], 0);
```
Let's do away with the intermediate variables and use the indexes directly. Makes it easier to replace them with constants via search-and-replace down the road.